source/unified-test-format/unified-test-format.md

```diff
@@ -0,0 +1,3459 @@
+
+Original GridFS spec tests refer to "upload", "download", and "download_by_name" methods, which allow the tests to
+abstract stream IO and either upload a byte sequence or assert a downloaded byte sequence. These operations correspond
+to the [download](#download), [downloadByName](#downloadByName), [upload](#upload-and-uploadWithId), and
+[uploadWithId](#upload-and-uploadWithId) [Bucket Operations](#bucket-operations).
+
+In order to support methods such as `downloadToStream`, `openUploadStream`, and `openUploadStreamWithId`, test runners
+would need to represent streams as entities and support IO operations to directly read from and write to a stream
+entity. This may not be worth the added complexity if the existing operations provide adequate test coverage for GridFS
+implementations.
+
+### Support Client-side Encryption integration tests
+
+Supporting client-side encryption spec tests will require the following changes to the test format:
+
+- `json_schema` will need to be specified when creating a collection, via either the collection entity definition or
+  [initialData](#initialData).
+- `key_vault_data` can be expressed via [initialData](#initialData)
+- `autoEncryptOpts` will need to be specified when defining a client entity. Preparation of this field may require
+  reading AWS credentials from environment variables.
+
+The process for executing tests should not require significant changes, but test files will need to express a dependency
+on mongocryptd.
+
+### Incorporate referenced entity operations into the schema version
+
+The [Schema Version](#schema-version) is not impacted by changes to operations defined in other specs and referenced in
+`Entity Test Operations` (e.g. `find` for CRUD). The [operation.name](#operation_name) and
+[operation.arguments](#operation_arguments) fields are loosely defined in the JSON schema as string and object types,
+respectively.
+
+Ideally, all operations (and their arguments) would be enforced by the JSON schema *and* any changes to operations would
+affect the schema version accordingly. For example, a new `find` option would warrant a minor version bump both for the
+CRUD spec and this spec and its schema.
+
+As discussed in [Executing an Operation](#executing-an-operation), test runners MUST raise errors for unsupported
+operations and arguments. This is a concession until such time that better processes can be established for versioning
+other specs *and* collating spec changes developed in parallel or during the same release cycle.
+
+## Changelog
+
+- 2024-02-06: Migrated from reStructuredText to Markdown.
+
+- 2024-01-17: **Schema version 1.19.**\
+  Add `authMechanism` to `runOnRequirement` and require that `uriOptions` supports
+  placeholder documents.
+
+- 2024-01-11: **Schema version 1.18.**\
+  Allow named KMS providers in `kmsProviders`. Note location of Client-Side
+  Encryption test credentials.
+
+- 2024-01-03: Document server version requirements for `errorLabels` and\
```

Comment:
https://spec.commonmark.org/0.29/#example-631 suggests the trailing slash is used for a hard line break. I can see doing that for the "Schema version" highlights (although the bold text should already make them stand out), but why here and other entries with non-formatted text?

Reply:
I'm not sure why pandoc chose to do that